flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackend.java

```diff
@@ -313,10 +317,16 @@ public void dispose() {
 		IOUtils.closeQuietly(dbOptions);
 		IOUtils.closeQuietly(columnOptions);
 
+		cleanInstanceBasePath();
```

Comment:
This again would not need the existence check that runs inside the method, because at this point the directory should always exist.

Reply:
addressed